lib/content-services/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@alfresco/adf-content-services",
   "description": "Alfresco ADF content services",
-  "version": "7.0.0-alpha.6",
+  "version": "7.0.0-alpha.7-0",
```

Comment:
In the same file update the js-api and adf dependencies version to `8.0.0-alpha.7-0` and `7.0.0-alpha.7-0`

Reply:
Same for dependencies in other package.json files